doc/contributing.md

```diff
@@ -488,10 +488,10 @@ integration systems will warn us and you can make a new commit with the formatte
 Even better, you can just write `/format` in the first line of any comment in a
 pull request to lint the code automatically.
 
-When wrapping a new alias, use an underscore to separate words bridged by vowels
-(aeiou), such as `no_skip` and `z_only`. Do not use an underscore to separate
-words bridged only by consonants, such as `distcalc`, and `crossprofile`. This
-convention is not applied by the code checking tools, but the PyGMT maintainers
+When wrapping a new alias, use an underscore to separate words. This improves the
```

Comment:
The convention should be applied to parameters of all functions，not just aliases